frontend/src/components/search-page/search-page.tsx

```diff
@@ -21,7 +27,26 @@ const SearchPage: React.FC = () => {
         <Title element="h3">Search</Title>
         <Grid container spacing={2}>
           <Grid item xs={12} md={8}>
-            {cars.length === 0 && <p>No cars found.</p>}
+            {cars.length === 0 && (
+              <>
+                <Container sx={{ display: 'flex' }}>
+                  <SearchIcon
+                    className={clsx(styles.searchIcon, styles.icon)}
+                  />
+                  <h3 className={styles.noCarsTitle}>No cars found</h3>
+                </Container>
+                <Container className={styles.topAutoriaContainer}>
+                  <div>
+                    <div className={styles.topAutoriaHeader}>Top Autoria</div>
+                    <div className={styles.topAutoriaCards}>
+                      {topCars.map((car) => (
+                        <TopCarCard car={car} key={car.id} />
+                      ))}
+                    </div>
```

Comment:
Let's create a component that wraps the top cars. So we can use it both here and on the landing